src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropertyPageSerializationService_Store.vb

```diff
@@ -459,7 +459,6 @@ Namespace Microsoft.VisualStudio.Editors.PropPageDesigner
 
                     If TypeOf Member Is PropertyDescriptor Then
                         Dim Prop As PropertyDescriptor = DirectCast(Member, PropertyDescriptor)
-                        Debug.Assert(Prop.PropertyType.IsSerializable)
```

Comment:
This code assumes that the property type that comes in contains the Serializable Attribute.The EnumProperty that backs up these enum properties return `IEnumValue`, which is an interface, cannot contain the Serializable Attribute. Hence this check is invalid at this point.